README.md

```diff
@@ -24,10 +24,16 @@ This module provides Cloud Trace support for Node.js applications. [Google Cloud
 
         npm install --save @google/cloud-trace
 
-2. Include the library at the *top of the main script of your application*. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
+2. Include the library by requiring it as part of your start up command (your project id must be specified using a config file or environment variable, see below):
+
+        node --require @google/cloud-trace app.js
+
+or at the *top of the main script of your application*:
 
         require('@google/cloud-trace').start({projectId: 'your-project-id'});
 
+It's important that Cloud Trace is the first thing executed so that it can accurately gather data. If you require multiple modules as part of your start up command, cloud trace must be the first since they are loaded in order.
+
```

Comment:
I think it would be better to state this *before* the mechanism to load is described.